content/en/docs/concepts/overview/working-with-objects/field-selectors.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Field Selectors
+weight: 60
+---
+
+_Field selectors_ enable you to [select Kubernetes resources](/docs/concepts/overview/working-with-objects/kubernetes-objects) on the basis of the value of one or more resource fields. Here are some example field selector queries:
```

Comment:
From the [style guide](https://kubernetes.io/docs/home/contribute/style-guide/#content-best-practices): use simple and direct language.
```
_Field selectors_ let you [select Kubernetes resources](/docs/concepts/overview/working-with-objects/kubernetes-objects) based on the value of one or more resource fields. Here are some sample field selector queries:
```
So:
> _Field selectors_ let you [select Kubernetes resources](/docs/concepts/overview/working-with-objects/kubernetes-objects) based on the value of one or more resource fields. Here are some sample field selector queries: